src/target/riscv/riscv.c

```diff
@@ -4255,6 +4289,57 @@ COMMAND_HANDLER(riscv_exec_progbuf)
 	return ERROR_OK;
 }
 
+COMMAND_HANDLER(riscv_eq_match_trigger_enable)
+{
+	struct target *target = get_current_target(CMD_CTX);
+	RISCV_INFO(r);
+
+	if (CMD_ARGC == 0) {
+		command_print(CMD, "equality match trigger enabled = %d", r->enable_equality_match_trigger);
+		return ERROR_OK;
+	} else if (CMD_ARGC != 1) {
+		LOG_ERROR("Command takes 0 or 1 parameters");
+		return ERROR_COMMAND_SYNTAX_ERROR;
+	}
+
+	COMMAND_PARSE_ON_OFF(CMD_ARGV[0], r->enable_equality_match_trigger);
+	return ERROR_OK;
```

Comment:
Please, use the below structure, as already recommended in the other merge request: https://github.com/riscv/riscv-openocd/pull/918#discussion_r1332496576

```
	if (CMD_ARGC == 0) {
		/* Do something */
	} else if (CMD_ARGC == 1) {
		/* Do something else */
	} else {
		/* Report error */
	}
```

Reply:
Or, maybe because of the checkpatch rules, this will be:

```
	if (CMD_ARGC == 0) {
		/* Do something */
	} else if (CMD_ARGC == 1) {
		/* Do something else */

	/* Report error */
```

Reply:
Addressed